### DISCUSSION TITLE
Noncommercial biomass

### DISCUSSION THREAD

**Discussion by jansteinhauser** [magpiemodel/magpie#301] [Q&A] [answered]
Hello again!

Since I didn't see it specifically mentioned anywhere, quick question about noncommercial biomass demands (e.g. collected branches and forest residues, dung fuels): Is MAgPIE taking those into account and, if so, how? 

Thank you!
Jan

**Comment by abhimishr** [comment, marked as answer]
Hi Jan,

What do you mean by non-commercial biomass? Would be nice if you can specify (or have an example). 

On forestry side - If you mean branches and residues - we do not have a demand for them per-se. Residues **can** be produced during forest harvest and removed from forest to potentially meet a portion of traditional wood fuel demand. 

If this is not what you are looking for then maybe an example of non-commercial biomass helps and then another MAgPIE member can answer this better.

**Reply by jansteinhauser** [reply]
Hi Abhijeet,

Thanks for your response and sorry for not being super clear in my question. You got it right, though, branches and other forest residues would have been my first example. Another one would be dung-based fuel. (Edited the initial question to clarify)

If this is not yet covered (sufficiently), we may look into how to add this in the future.

**Reply by abhimishr** [reply]
Hi Jan,

This sounds like an interesting proposition indeed! Can you also drop a line to magpie@pik-potsdam.de regarding this? Then it makes it way to the internal discussions and we can start the ball rolling in terms of brain storming.

@tscheypidi - How do we handle feature request/propositions at the moment?

cheers,
Abhi

**Reply by jansteinhauser** [reply]
Hi Abhi,

Thanks, great to hear that this may be of interest to you as well. I'll discuss it again with my IIASA colleague before "officially" proposing it via mail. We have some experts on the topic in our program here and could potentially work on this. Just to be clear, I didn't mean it as a feature request that we'd hope for PIK to work on (alone) - just in case that's how my comment is understood.

Thanks again for your quick replies!
Jan

**Reply by tscheypidi** [reply]
Sounds cools. @abhimishr Concerning your question about feature requests. I believe in most cases it would mean that we would be open to review pull requests implementing such features and integrating it into the model. And with Jan on board I could imagine that we could get such a cool feature as PR if it makes sense for IIASA, right Jan? :)

**Reply by jansteinhauser** [reply]
Cool. We're definitely interested in having that in the model. I'll discuss it with the IIASA people and will definitely ping you about input on this in the future. So, it's probably more of a question of when than if. :)
